doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,322 @@
+
+
+Click on **Finish**.
+#### 
+
+<br>
+#### Adding {{%tts%}} Server Address in the Gateway
+<br>
+
+- Click on "**LoRaWAN**" in the menu on the left. It opens the Gateway's configuration page.
+
+<br>
+
+{{< figure src="017_001_Gateway_Menu_LoRaWAN.png" alt="Gateway Menu" >}}
+<br>
+
+- Select the mode as "**Packet Forwarder**" 
+{{< figure src="018_Packet_Forwarder_Gateway.png" alt="Packet Forwarder Gateway" >}}
+
+<br>
+
+- Select your **frequency plan** (EU868 in this case.)
+
+{{< figure src="019_Frequency_Plan.png" alt="Frequency Plan" >}}
+
+- Scroll down and add:
+
+|---------------------------------------------------------------------|-----------------------------------------|
+| {{%tts%}} URL provided to you as the **Server Address**. | `thethings.example.com` |
+| Upstream Port                                                       | 1700                                    |
+| Downstream Port                                                     | 1700                                    |
+
+
+> If you followed the **[Getting Started guide](https://enterprise.thethingsstack.io/v3.4.0/guides/getting-started/)** , the domain of {{%tts%}} is the same as what you use instead of **https://thethings.example.com**. For example, \<domain\>.eu1.cloud.thethings.industries
+
+{{< figure src="020_Server_Setup_Configuration.png" alt="Server Configuration" >}}
+
+
+- Click on **Submit**.
+
+<br>
+
+
+#### 
+
+
+#### Network Interfaces
+
+Network interface shows the configuration for one or more network interfaces available.
+
+
+- Select **Setup > Network Interfaces** from the left menu:
+
+{{< figure src="021_Network_Interfaces.png" alt="Network Interfaces" >}}
+<br>
+
+- Click on the pencil icon(edit) in the "Options" column corresponding to **eth0**. You will be redirected to the Network Interface Configuration window. 
+<br>
+{{< figure src="022_Network_Interface_Configuration_Window.png" alt="Network Interface Configuration" >}}
+<br>
+
+- In the configuration window, change the **Bridge** from **br0** to **--**.
+- In IPv4 Settings, change the **Mode** from **Static** to **DHCP Client**.
+- Click on **Submit**.
+- Now click on **Save and Restart** from the menu.
+
+{{< figure src="017_002_Gateway_Menu_Save-And-Restart.png" alt="Gateway Menu" >}}
+
+<br>
+
+- You will be prompted to confirm the restart. Choose **OK** to proceed.
+
+{{< figure src="023_System_Confirmation_Prompt.png" alt="System Confirmation" >}}
+
+
+- Disconnect the temporary network cable and connect the Conduit to the target network while it restarts.
+
+To know more about other features of the MultiTech Conduit gateway, you can refer to this **[software guide](https://www.multitech.com/documents/publications/software-guides/s000727%20-%20mPower%20Edge%20Intelligence%20Conduit_AEP_software_guide.pdf).**
+
+<br>
+
+## Upgrading the Firmware
```

Comment:
This section doesnt make sense here. We can include it in trouble shooting/FAQs section. Check how they structured in TTN public documentation.